packages/transport/src/transports/abstractApi.ts

```diff
@@ -187,51 +193,40 @@ export abstract class AbstractApiTransport extends AbstractTransport {
                 }
                 const { path } = getPathBySessionResponse.payload;
 
-                try {
```

Comment:
try catch is not needed here, the function body is wrapped into custom schedule action which has its own catch handler.